app/services/hmpps_api/community_api.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module HmppsApi
+  class CommunityApi
+    def self.client
+      host = Rails.configuration.community_api_host
+      HmppsApi::Client.new(host + '/secure')
+    end
+
+    def self.set_pom(offender_no:, prison:, forename:, surname:)
+      safe_offender_no = URI.encode_www_form_component(offender_no)
+      route = "/offenders/nomsNumber/#{safe_offender_no}/prisonOffenderManager"
+
+      body = {
+        nomsPrisonInstitutionCode: prison,
+        officer: {
+          forenames: forename,
+          surname: surname
+        }
+      }
+
+      # We don't use a cache here because we're not 'getting' data, but 'pushing' it
+      self.client.put(route, body)
+
+      # So long as the API call didn't error, return true
+      # The API response body isn't useful to us
+      true
```

Comment:
unsure about always returning true here - implies that you can somehow check the response for true/false. Probably more idiomatic not to return anything - it will probably be 'truthy' but clients shouldn't rely on it.